### ISSUE TITLE
How to resolve warning: Can only update a mounted or mounting component for Array

### ISSUE THREAD

**Issue by kavink** [final-form/react-final-form-arrays#21]
Any thoughts on how I can resolve this ? The FieldArray field is just function. And using that in my form.
```
 Warning: Can only update a mounted or mounting component. This usually means you called setState, replaceState, or forceUpdate on an unmounted component. This is a no-op.

Please check the code for the ReactFinalFormFieldArray(4.2.0)(undefined) component.

```

**Comment by heikki** [comment]
Same problem here. I haven't been able to reproduce it in a simple example.
It happens only when there's a validation in field array field and it seems to originate from a setTimeout call here:

https://github.com/final-form/react-final-form-arrays/blob/c44ad1d223cbfedc2631cf15a558132d4ceddc61/src/FieldArray.js#L86

**Comment by kavink** [comment]
@heikki Yes , it also causes performance impact as mentioned in #22

**Comment by erikras** [comment]
We might have to [keep track of whether or not the component is still mounted](https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html).

**Comment by kavink** [comment]
@erikas  a small snippet on how to do Woolf be great , I’m assuming I should check for my full form or just array fields ?  Thanks !

**Comment by heikki** [comment]
Great! Fix seems to work. Can we get a new release or is it waiting for something else?

**Comment by erikras** [comment]
Published fix in [`v1.0.4`](https://github.com/final-form/react-final-form-arrays/releases/tag/v1.0.4).
